tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+        "tslint:recommended"
+    ],
+    "jsRules": {},
+    "rules": {},
```

Comment:
I like single quotes

Reply:
This file is autogenerated by tslint init.
it also lint the code to use double quotes.
I do not care ..

Reply:
```
"rules": {
  "quotemark": [true, "single"]
}
```

Reply:
lets argue about that :)

Reply:
single quotes don't required pressing `shift`. Boom!